content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -33,6 +33,7 @@ Kubernetes as a project currently supports and maintains [GCE](https://git.k8s.i
 * F5 Networks provides [support and maintenance](https://support.f5.com/csp/article/K86859508)
   for the [F5 BIG-IP Controller for Kubernetes](http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest).
 * [Gloo](https://gloo.solo.io) is an open-source ingress controller based on [Envoy](https://www.envoyproxy.io) which offers API Gateway functionality with enterprise support from [solo.io](https://www.solo.io).  
+* [HAProxy Ingress](https://haproxy-ingress.github.io) is a highly customizable community-driven ingress controller for HAProxy.
 * [HAProxy Technologies](https://www.haproxy.com/) offers support and maintenance for the [HAProxy Ingress Controller for Kubernetes](https://github.com/haproxytech/kubernetes-ingress). See the [official documentation](https://www.haproxy.com/documentation/hapee/1-9r1/traffic-management/kubernetes-ingress-controller/).
```

Comment:
It looks like this line is covering the same topic.

Reply:
Hi @sftim thanks for reviewing!

They are distinct products. While Ingress from HAProxy Tech is the HAProxy Tech supported one, HAProxy Ingress is a community driven one. Although HAProxy Tech staff helped to evolve HAProxy Ingress in the past, now they even doesn't share the same code base.